app.js

```diff
@@ -0,0 +1,157 @@
+//jshint esversion: 6
+const express = require("express");
+const bodyParser = require("body-parser");
+const mongoose = require("mongoose");
+const redisClient = require("redis").createClient;
+const redis = redisClient(6379, 'redis-server');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
+app.use(express.static("public"));
+//let studentMap = new Map();
+
+mongoose.connect("mongodb://mongo:27017/mongorediscomposeDB", {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  useFindAndModify: false
+}, function(err) {
+  if (!err) {
+    console.log("successfully connected");
+  }
+});
+
+redis.on("connect", function() {
+  console.log('connected to Redis');
+});
+
+app.listen(3000, function() {
+  console.log("Server started at https://localhost:3000");
+});
+
+const studentSchema = mongoose.Schema({
```

Comment:
Usually the convention of defining entities / Schema is to begin with a Capital Letter, but not sure if that follows here as well